cinema/views.py

```diff
@@ -56,3 +91,56 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+    def get_queryset(self):
+        queryset = self.queryset
+        date = self.request.query_params.get("date")
+        movie_id = self.request.query_params.get("movie")
+
+        if date:
+            year, month, day = date.split("-")
+            queryset = queryset.filter(
+                show_time__year=year,
+                show_time__month=month,
+                show_time__day=day
+            )
```

Comment:
Not a mistake, but could also use strptime().date() and than compare show_time__date.

Reply:
Yeah, Dmytro is correct. But you can also just use show__time__date = date. Django should understand this string in this format